app/Console/Commands/UpdateEnquiryThreadsGat7178.php

```diff
@@ -0,0 +1,141 @@
+                "id"  => 142,
+                "user_id"  => 1112,
+                "team_ids"  => [21, 119],
+                "project_title"  => "",
+                "unique_key"  => "YWxdOyyr",
+                "is_dar_dialogue"  => 0,
+                "is_dar_status"  => 0,
+                "enabled"  => 1,
+                "is_general_enquiry"  => 1,
+                "is_feasibility_enquiry"  => 0,
+                "is_dar_review"  => 0,
+                "created_at"  => null,
+                "updated_at"  => null
+            ],
+            [
+                "id"  => 148,
+                "user_id"  => 4608,
+                "team_ids"  => [48, 14, 45, 19],
+                "project_title"  => "Accelerating AI-Driven Clinical Tools Using Real-World Health Data from Acute and Chronic Care Settings",
+                "unique_key"  => "iBWfwKX2",
+                "is_dar_dialogue"  => 0,
+                "is_dar_status"  => 0,
+                "enabled"  => 1,
+                "is_general_enquiry"  => 0,
+                "is_feasibility_enquiry"  => 1,
+                "is_dar_review"  => 0,
+                "created_at"  => null,
+                "updated_at"  => null
+            ],
+        ];
+
+        foreach ($array as $value) {
+            $this->info("Processing enquiry thread with ID: {$value['id']}");
+
+            $id = $value['id'];
+            $teamIds = $value['team_ids'];
+
+            foreach ($teamIds as $teamId) {
+                $this->info("Processing team ID: {$teamId}");
+
+                $check = EnquiryThread::where([
+                        'id' => $id,
+                        'team_id' => $teamId,
+                    ])
+                    ->first();
+
+                if (!is_null($check)) {
+                    $this->warn("Enquiry thread with ID: {$id} and team ID: {$teamId} already exists. Skipping.");
+                    continue;
+                }
+
+                $newEnquiryThread = EnquiryThread::create([
+                    'id' => $id,
+                    'user_id' => $value['user_id'],
+                    'team_id' => $teamId,
+                    'project_title' => $value['project_title'],
+                    'unique_key' => $value['unique_key'],
+                    'enquiry_unique_key' => $value['unique_key'],
+                    'is_dar_dialogue' => $value['is_dar_dialogue'],
+                    'is_dar_status' => $value['is_dar_status'],
+                    'enabled' => $value['enabled'],
+                    'is_general_enquiry' => $value['is_general_enquiry'],
+                    'is_feasibility_enquiry' => $value['is_feasibility_enquiry'],
+                    'is_dar_review' => $value['is_dar_review'],
+                ]);
+                $this->info("Created new enquiry thread with ID: {$newEnquiryThread->id} for team ID: {$teamId} and for original ID: {$id}");
+
+                $messages = EnquiryMessage::where('thread_id', $newEnquiryThread->id)->get();
```

Comment:
dont get it. There's a new enquiry thread that has been created - so messages is always going to be empty?

Reply:
updated